setup.py

```diff
@@ -6,7 +6,7 @@
 
 
 install_requires = ['async-timeout']
-if platform.python_implementation() == 'CPython':
+if platform.python_implementation() == 'CPython' and not os.environ.get("WITHOUT_HIREDIS"):
```

Comment:
Maybe "hiredis" should be moved into an `extras_require`?
(I assume this is not due to the Python version, but because hiredis should be skipped/is not available in general)